api/buckets/service.go

```diff
@@ -82,16 +82,20 @@ func (s *Service) createBucket(ctx context.Context, dbID thread.ID, dbToken thre
 		ipfsfiles.NewMapDirectory(map[string]ipfsfiles.Node{
 			seedName: ipfsfiles.NewBytesFile(seed),
 		}),
-		options.Unixfs.Pin(true))
+		options.Unixfs.Pin(false))
```

Comment:
Any words on this change?
Seems like as is no Cids are pinned. I'm not sure how the Pin update behaves if its moving a pin which wasn't `.Pin(true)`. Also, if the IPFS node runs GC before a further PushPath is done, we'll lose the bucket.

Reply:
The problem with pinning here is that the Cid V0 is pinned, and then below when we update it, it will error w/ "'from' cid was not recursively pinned already", since we'll be asking about the Cid V1. Instead, we add it, then on the next line, manually (recursively) pin the Cid in V1 form.

Reply:
Ah, I see. Right, the ipfs-node is talking about V0, and you want to pin V1.
Uhm, how unfortunate. It would be good that the ipfs-node can see that both Cids are referring to the same sha256 data. I missed the `.Pin().Add()` below, so looks good.

Reply:
Oh, actually looks like there a cid version option with add... I'll try that.

Reply:
Sounds nice!

Reply:
Well damn, looks like I can ditch that util method all together after specifying the cid version with add.

Reply:
Nice! :)